kobo/apps/service_health/test_service_health_minimal.py

```diff
@@ -1,14 +1,14 @@
-from django.test import TestCase
-from django.urls import reverse
+from django.test import TestCase, override_settings
 
 
 class ServiceHealthMinimalTestCase(TestCase):
-    url = reverse('service-health-minimal')
+    url = '/service_health/minimal/'
```

Comment:
I would leave a comment here why we do not (cannot) use `reverse()` (e.g.: Because it does not belong to `url_patterns`) . What do you think?